fe/src/main/java/org/apache/doris/persist/EditLog.java

```diff
@@ -378,8 +379,8 @@ public static void loadJournal(Catalog catalog, JournalEntity journal) {
                     break;
                 case OperationType.OP_FINISH_SYNC_DELETE: {
                     DeleteInfo info = (DeleteInfo) journal.getData();
-                    Load load = catalog.getLoadInstance();
-                    load.replayDelete(info, catalog);
+                    DeleteHandler deleteHandler = catalog.getDeleteHandler();
```

Comment:
You can not use the origin `OP_FINISH_SYNC_DELETE`, because they are different operations.

Reply:
Now add a new operation OP_FINISH_DELETE